server.coffee

```diff
@@ -12,6 +12,7 @@ configWrapper = require './lib/configWrapper'
 load = require './lib/load'
 
 MODULES = config.modules
+ALLOW_ORIGIN_HEADERS = config.http_headers
```

Comment:
If the option field is going to accept arbitrary headers, it seems like this should be named `HEADERS` or the like, rather than `ALLOW_ORIGIN_HEADERS`.  It also seems like it should load all the headers and apply them, rather than pulling out a single header on line 62.